package.json

```diff
@@ -206,7 +207,7 @@
     "test": "NODE_ENV=test LOG_LEVEL=silent mocha './app/**/*.test.js' './common/**/*.test.js' './config/**/*.test.js'",
     "test-e2e:browserstack:ie": "testcafe 'browserstack:IE@11.0:Windows 10' test/e2e/*.test.js",
     "test-e2e:chrome": "testcafe chrome test/e2e/*.test.js",
-    "test-e2e:ci": "npm run test-e2e:chrome -- --color --reporter spec,xunit:reports/testcafe/results-chrome.xml",
+    "test-e2e:ci": "node test/e2e/parallel-runner.js",
```

Comment:
It might be nice to be able to pass commands into the parallel runner, for example the reporter or browser to use. That would mean we could use it when running tests locally too...